src/style.css

```diff
@@ -1 +1,89 @@
-/* Styles go here */
+html {
+  scroll-behavior: smooth;
+  box-sizing: border-box;
+  overflow: hidden;
+}
+
+ul,
+li,
+a {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --color-active: #00acdc;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.header {
```

Comment:
Add padding: 0 50px; o your elements like header__logo and nav__menu will not need margins